standalone-node/docs/user-guide/desktop-virtualization-image-guide.md

```diff
@@ -218,11 +230,14 @@ runcmd:
   # Source /etc/environment to ensure newly created environment variables are available to subsequent commands in this boot sequence
   - source /etc/environment
   - udevadm control --reload-rules
+  # Add the user to render group (assuming username is 'user')
+  - sudo usermod -a -G render user
+  - sudo -u user mkdir -p /home/user/.config/openbox/
+  - sudo -u user mv /etc/cloud/rc.xml /home/user/.config/openbox/rc.xml
   # Change `guest` to your intended username if not using 'guest' user.
-  - sudo -u guest mkdir -p /home/guest/.config/openbox/
-  - sudo -u guest mv /etc/cloud/rc.xml /home/guest/.config/openbox/rc.xml  
-  - sudo -u guest XDG_RUNTIME_DIR=/run/user/$(id -u guest) systemctl --user enable idv-init.service
-  - sudo -u guest XDG_RUNTIME_DIR=/run/user/$(id -u guest) systemctl --user start idv-init.service
-  - test -f /opt/user-apps/network_config.sh && bash /opt/user-apps/network_config.sh /etc/cloud/custom_network.conf || echo "network_config.sh is missing"
-  - test -f /opt/user-apps/apply_bridge_nad.sh && bash /opt/user-apps/apply_bridge_nad.sh /etc/cloud/custom_network.conf > /etc/cloud/apply_bridge_nad.log 2>&1 &  
+  - sudo -u user XDG_RUNTIME_DIR=/run/user/$(id -u user) systemctl --user enable idv-init.service
+  - sudo -u user XDG_RUNTIME_DIR=/run/user/$(id -u user) systemctl --user start idv-init.service
+  - sudo systemctl start nw_custom_file.service
+  - test -f /opt/user-apps/scripts/management/apply_bridge_nad.sh && bash /opt/user-apps/scripts/management/apply_bridge_nad.sh /etc/cloud/custom_network.conf > /etc/cloud/apply_bridge_nad.log 2>&1
+  - bash /opt/user-apps/scripts/management/app-deploy.sh
```

Comment:
app-deploy.sh is not available in the repo... This line should be removed.. Generic customer will add their application specific commands to automate the application deployment. Just add a comment here.